contracts/bonding/BondingManager.sol

```diff
@@ -363,10 +352,8 @@ contract BondingManager is IBondingManager, Manager {
         // Set last round that transcoder called reward
         transcoders[msg.sender].lastRewardRound = currentRound;
 
-        // Calculate number of tokens to mint
-        uint256 mintedTokens = mintedTokensPerReward(msg.sender);
-        // Mint token reward and allocate to this protocol contract
-        token.mint(this, mintedTokens);
+        // Mint token reward
+        uint256 mintedTokens = minter().mint(activeTranscoders[activeTranscoderPositions[msg.sender]].key, totalActiveTranscoderStake);
```

Comment:
With the addition of the Minter contract I thought it might make sense to move the logic for calculating how many new tokens to mint into the Minter contract. Now, the BondingManager tells the Minter what the current transcoder's stake and the total active transcoder stake is. The Minter then uses that information to decide how many tokens to mint.